exaudfclient/base/python/extend_exascript_python_preset_py.sh

```diff
@@ -1,3 +1,5 @@
+#!/bin/bash
```

Comment:
Reason:
^-- SC2148: Tips depend on target shell and yours is unknown. Add a shebang.

Reply:
yup thats fine